js/src/tooltip.js

```diff
@@ -283,6 +283,10 @@ class Tooltip extends BaseComponent {
 
     EventHandler.trigger(this._element, this.constructor.Event.INSERTED)
 
+    if (this._popper) {
+      this._popper.destroy()
+    }
+
```

Comment:
Instead of destroying the previous popper instance and then creating a new one, you can just use the previous instance if it exists :thinking: 

```js
if (!this._popper) {
  this._popper = Popper.createPopper(this._element, tip, this._getPopperConfig(attachment))
}
```

Reply:
yes, with one caveat, the existing popper needs to be updated in case the the tooltip text has changed (which means it needs to be repositioned). This is applicable to the "copy to clipboard" buttons in the documentation for example. I've applied this change.